packages/mobile/src/components/rate-cta-drawer/RateCtaReminder.tsx

```diff
@@ -11,6 +11,9 @@ import { requestReview } from 'app/store/rate-cta/slice'
 
 const getHasAccount = accountSelectors.getHasAccount
 
+const FIRST_REMINDER_SESSION = 3
+const REMINDER_EVERY_N_SESSIONS = 5
```

Comment:
maybe use "frequency", since i think that's the name of the param? so like reminderFrequency?